### PR TITLE
Feature: Add ability to control zeplin overlay with Shift + arrow-keys

### PR DESCRIPTION
**Pull request by lysenko-sergey-developer**
Hi @mertkahyaoglu! Awesome, repository!

This PR adds the ability to control overlay with Shift + Arrow keys. This ability might be helpful when you want to do pixel perfect.

### CODE REVIEW

File: src/components/OverlayImage.tsx [hunk 1]
```diff
@@ -39,6 +39,28 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
     return () => window.removeEventListener("mousemove", handleMouseMove);
   }, [mouseDown]);
 
+  // Ability to move zeplin overlay with Shift + Arrow keys
```
Comment (mertkahyaoglu) [inline src/components/OverlayImage.tsx:42]: I think we need to show this in the UI but not sure where. Maybe we can mention this in readme.md later.

File: src/components/OverlayImage.tsx [hunk 2]
```diff
@@ -39,6 +39,28 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
...
+  useEffect(() => {
+    const handleKeyDown = (event) => {
```
Comment (mertkahyaoglu) [inline src/components/OverlayImage.tsx:44]: We need to respect `isLocked` here. If it is true, we shouldn't update position.

**Review by mertkahyaoglu — CHANGES_REQUESTED** [review]
@lysenko-sergey-developer Thanks! This is a really good addition.

I have two comments related to the changes.

**Comment by yannbf** [comment]
Hey @mertkahyaoglu there's a failure in the netlify preview. Is it related to this PR?

```
1:12:12 PM: ERR! Module not found: Error: Can't resolve '../dist/register' in '/opt/build/repo/.storybook'
ERR! ModuleNotFoundError: Module not found: Error: Can't resolve '../dist/register' in '/opt/build/repo/.storybook'
```

File: src/components/OverlayImage.tsx (hunk 1 above)
Comment (yannbf) [reply src/components/OverlayImage.tsx:42]: I was about to mention the same! Maybe a tooltip? small `?` icon somewhere that gives a list of helpful tips?

File: src/components/OverlayImage.tsx (hunk 2 above)
Comment (lysenko-sergey-developer) [reply src/components/OverlayImage.tsx:43]: Added check for ```isLocked``` property.

File: src/components/OverlayImage.tsx (hunk 1 above)
Comment (mertkahyaoglu) [reply src/components/OverlayImage.tsx:41]: Tooltip works but requires a bit much work for this pr. Not sure how to style it though. Another way could be showing a text info in the preview panel (at the bottom maybe) only when dragging the overlay. So it won't distract users all the time.

**Comment by mertkahyaoglu** [comment]
@yannbf I think it was a cache issue. Now looks good.

File: src/components/OverlayImage.tsx [hunk 3]
```diff
@@ -39,6 +38,30 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
...
+        updatePosition({ x: 0, y: -1 });
+      }
+      if (event.shiftKey && event.key === "ArrowDown") {
+        updatePosition({ x: 0, y: 1 });
+      }
+      if (event.shiftKey && event.key === "ArrowRight") {
+        updatePosition({ x: 1, y: 0 });
+      }
+      if (event.shiftKey && event.key === "ArrowLeft") {
+        updatePosition({ x: - 1, y: 0 });
```
Comment (mertkahyaoglu) [inline src/components/OverlayImage.tsx:54]:
```suggestion
        updatePosition({ x: -1, y: 0 });
```

File: src/components/OverlayImage.tsx [hunk 4]
```diff
@@ -39,6 +38,30 @@ const OverlayImage = ({ url, opacity, scaling, isLocked, showDifference }: Overl
     return () => window.removeEventListener("mousemove", handleMouseMove);
   }, [mouseDown]);
 
+  // Ability to move zeplin overlay with Shift + Arrow keys
```
Comment (mertkahyaoglu) [inline src/components/OverlayImage.tsx:41]: I think we can remove this comment. Also, I'm not used to seeing our company name lowercase 😬

**Review by mertkahyaoglu — COMMENTED** [review]
Minor comments and then good to go. Sorry I missed these in the first review.

**Review by mertkahyaoglu — APPROVED** [review]

**Comment by mertkahyaoglu** [comment]
Awesome addition! Thanks @lysenko-sergey-developer.

**Pull request merged**